ui/app/actions.js

```diff
@@ -705,11 +705,10 @@ function signTypedMsg (msgData) {
 
 function signTx (txData) {
   return (dispatch) => {
-    dispatch(actions.showLoadingIndication())
```

Comment:
Why do you remove `showLoadingIndication` and `hideLoadingIndication` here, but not in the other methods edited below?

Reply:
This is a weird case where the callback passed to `ethQuery.sendTransaction` gets called when the tx gets approved or rejected, and not when we'd expect (called after the tx is added to the list of unapproved tx'es). The loading spinner was being hidden unintentionally during approving/rejecting on the confirm step.